### ISSUE TITLE
matchbox RPM should create /var/lib/matchbox and /etc/matchbox directories

### ISSUE THREAD

**Issue by dghubble** [poseidon/matchbox#481]
Users who install the RPM from Copr should not need to create these directories or the subdirectories (profiles, groups, ignition, cloud, assets) manually.

EDIT: Directory permissions need to be suitable as well.

Report by @rothgar

**Comment by dghubble** [comment]
@rothgar On Fedora 25, I'm seeing the `/var/lib/matchbox` directory is being created, but not the child directories or `/etc/matchbox`.

```
rpm -ql matchbox
/usr/bin/matchbox
/usr/lib/systemd/system/matchbox.service
/usr/share/doc/matchbox
/usr/share/doc/matchbox/CHANGES.md
/usr/share/doc/matchbox/CONTRIBUTING.md
/usr/share/doc/matchbox/DCO
/usr/share/doc/matchbox/LICENSE
/usr/share/doc/matchbox/NOTICE
/usr/share/doc/matchbox/README.md
/var/lib/matchbox
```

**Comment by rothgar** [comment]
You're probably right. I'm not sure if matchbox is supposed to create the subdirectories. If so, it is probably just a permissions issue.

**Comment by dghubble** [comment]
Yeah, its desired that those directories get created. Users who install the RPM and create profiles, groups, etc. via the gRPC API (e.g. Tectonic or Terraform) shouldn't have issues because matchbox will make the directories with the correct permissions. But for users who add JSON objects manually, they would need to make the directory and that's undesired.

I messed around with trying to fix this before the v0.6.0 RPM release, but ended up making it worse and reverting to get it out in time. The RPM spec is added in #496 and I'd like to find the right way to do this and put out a v0.6.1 if possible.

**Comment by dghubble** [comment]
cc @brianredbeard
